src/workers/notifications/postAdded.ts

```diff
@@ -61,6 +68,19 @@ const worker: NotificationWorker = {
             } as NotificationPostContext & Partial<NotificationDoneByContext>,
           }),
         );
+
+        if (post.type === PostType.Share) {
+          // squad_post_live notification
+          if (baseCtx.sharedPost?.origin === PostOrigin.Squad) {
```

Comment:
The article post is the one that has `squad` origin though.
Shared post will always have `user_generated`

Think we need to listen to the original post, but send the linked squad post?

Reply:
Hmm think you do have it, might be confusing to call it `sharedPost`?

Reply:
`sharedPost` is the post that is referenced hence the article post. It's something baked in our notification context. We have `post` and `sharedPost`. Agree that it might be confusing, happy to change it to something else

Reply:
Ah i see its a "notification shared post" not actual "SharePost" 😅
Hmm not sure what the alternative is though, but if we have some better ones I'd love to see that.